.github/workflows/nim_test.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: nim_test
+
+# yamllint disable-line rule:truthy
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  nim_test:
+    runs-on: ${{matrix.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
Why not use `ubuntu-latest` directly?

Reply:
To make it easier in the future to introduce other operating systems.

Reply:
Why not testing it directly against Windows and MacOS ?

Reply:
Done in de0c8d2bf332623f5f78dbacd33d3822472f52de.